crash on new build

// Don't do that.

Time: 09/06/15 21:28
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: li/cil/oc/api/driver/Block
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:538)
	at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoClassDefFoundError: li/cil/oc/api/driver/Block
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2663)
	at java.lang.Class.getConstructor0(Class.java:3067)
	at java.lang.Class.newInstance(Class.java:408)
	at pneumaticCraft.common.thirdparty.ThirdPartyManager.index(ThirdPartyManager.java:80)
	at pneumaticCraft.PneumaticCraft.PreInit(PneumaticCraft.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:535)
	... 10 more
Caused by: java.lang.ClassNotFoundException: li.cil.oc.api.driver.Block
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 42 more
Caused by: java.lang.NullPointerException

here is the full report no open computers 


---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 10/06/15 16:07
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: li/cil/oc/api/driver/Block
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:538)
	at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoClassDefFoundError: li/cil/oc/api/driver/Block
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2663)
	at java.lang.Class.getConstructor0(Class.java:3067)
	at java.lang.Class.newInstance(Class.java:408)
	at pneumaticCraft.common.thirdparty.ThirdPartyManager.index(ThirdPartyManager.java:80)
	at pneumaticCraft.PneumaticCraft.PreInit(PneumaticCraft.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:535)
	... 10 more
Caused by: java.lang.ClassNotFoundException: li.cil.oc.api.driver.Block
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 42 more
Caused by: java.lang.NullPointerException


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 7 (amd64) version 6.1
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 369176744 bytes (352 MB) / 1818755072 bytes (1734 MB) up to 3817865216 bytes (3641 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4096m -Xms256m -XX:PermSize=256m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.3.1428 Optifine OptiFine_1.7.10_HD_B7 179 mods loaded, 179 mods active
	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.3.1428-1.7.10.jar) Unloaded->Constructed->Pre-initialized
	Forge{10.13.3.1428} [Minecraft Forge] (forge-1.7.10-10.13.3.1428-1.7.10.jar) Unloaded->Constructed->Pre-initialized
	AM2-Preloader{0.0.2} [AMCore] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	appliedenergistics2-core{rv2-stable-6} [AppliedEnergistics2 Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	Aroma1997Core{1.0.2.13} [Aroma1997Core] (Aroma1997Core-1.7.10-1.0.2.13.jar) Unloaded->Constructed->Pre-initialized
	CodeChickenCore{1.0.6.43} [CodeChicken Core] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	InfiniBows{1.3.0 build 20} [InfiniBows] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	MobiusCore{1.2.5} [MobiusCore] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	NotEnoughItems{1.0.4.107} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.4.107-universal.jar) Unloaded->Constructed->Pre-initialized
	VeinMiner_coremod{0.27.1_build.unknown} [Core mod] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	OpenModsCore{0.7.3} [OpenModsCore] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) Unloaded->Constructed->Pre-initialized
	debug{1.0} [debug] (denseores-1.6.2.jar) Unloaded->Constructed->Pre-initialized
	AnimationAPI{1.2.4} [AnimationAPI] (AnimationAPI-1.7.10-1.2.4.jar) Unloaded->Constructed->Pre-initialized
	arsmagica2{1.4.0.008} [Ars Magica 2] (1.7.10_AM2-1.4.0.008.jar) Unloaded->Constructed->Pre-initialized
	IC2{2.2.727-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.727-experimental.jar) Unloaded->Constructed->Pre-initialized
	advancedgenetics{1.5.7} [Advanced Genetics] (advancedgenetics-1.7.10-1.5.7.jar) Unloaded->Constructed->Pre-initialized
	AppleCore{1.1.0} [AppleCore] (AppleCore-mc1.7.10-1.1.0.jar) Unloaded->Constructed->Pre-initialized
	appliedenergistics2{rv2-stable-6} [Applied Energistics 2] (appliedenergistics2-rv2-stable-6.jar) Unloaded->Constructed->Pre-initialized
	Aroma1997CoreHelper{1.0.2.13} [Aroma1997Core|Helper] (Aroma1997Core-1.7.10-1.0.2.13.jar) Unloaded->Constructed->Pre-initialized
	AromaBackup{0.0.0.5} [AromaBackup] (AromaBackup-1.7.10-0.0.0.5.jar) Unloaded->Constructed->Pre-initialized
	CoFHCore{1.7.10R3.0.2} [CoFH Core] (CoFHCore-[1.7.10]3.0.2-283.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Core{7.0.8} [BuildCraft] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	Mantle{1.7.10-0.3.2.jenkins184} [Mantle] (Mantle-1.7.10-0.3.2.jar) Unloaded->Constructed->Pre-initialized
	Natura{2.2.0} [Natura] (natura-1.7.10-2.2.0.1.jar) Unloaded->Constructed->Pre-initialized
	BiomesOPlenty{2.1.0} [Biomes O' Plenty] (BiomesOPlenty-1.7.10-2.1.0.1067-universal.jar) Unloaded->Constructed->Pre-initialized
	Forestry{3.6.0.586-unstable} [Forestry for Minecraft] (forestry_1.7.10-3.6.0.586-unstable.jar) Unloaded->Constructed->Pre-initialized
	bcapiHolder{1.0} [bcapiHolder] (bcapi-6.0.x.jar) Unloaded->Constructed->Pre-initialized
	bdlib{1.8.2.92} [BD Lib] (bdlib-1.8.2.92-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	BetterTitleScreen{1.7.10-1.1} [Better Title Screen] (BetterTitleScreen-1.7.10-1.1.jar) Unloaded->Constructed->Pre-initialized
	BiblioCraft{1.10.4} [BiblioCraft] (BiblioCraft[v1.10.4][MC1.7.10].jar) Unloaded->Constructed->Pre-initialized
	BiblioWoodsBoP{1.9} [BiblioWoods Biomes O'Plenty Edition] (BiblioWoods[BiomesOPlenty][v1.9].jar) Unloaded->Constructed->Pre-initialized
	BiblioWoodsForestry{1.7} [BiblioWoods Forestry Edition] (BiblioWoods[Forestry][v1.7].jar) Unloaded->Constructed->Pre-initialized
	BiblioWoodsNatura{1.5} [BiblioWoods Natura Edition] (BiblioWoods[Natura][v1.5].jar) Unloaded->Constructed->Pre-initialized
	ThermalFoundation{1.7.10R1.0.0} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.0.0-87.jar) Unloaded->Constructed->Pre-initialized
	ThermalExpansion{1.7.10R4.0.1} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.0.1-182.jar) Unloaded->Constructed->Pre-initialized
	BigReactors{0.4.3A} [Big Reactors] (BigReactors-0.4.3A.jar) Unloaded->Constructed->Pre-initialized
	BinnieCore{2.0-pre10} [Binnie Core] (binnie-mods-2.0-pre10.jar) Unloaded->Constructed->Pre-initialized
	Botany{2.0-pre10} [Botany] (binnie-mods-2.0-pre10.jar) Unloaded->Constructed->Pre-initialized
	ExtraBees{2.0-pre10} [Extra Bees] (binnie-mods-2.0-pre10.jar) Unloaded->Constructed->Pre-initialized
	ExtraTrees{2.0-pre10} [Extra Trees] (binnie-mods-2.0-pre10.jar) Unloaded->Constructed->Pre-initialized
	Genetics{2.0-pre10} [Genetics] (binnie-mods-2.0-pre10.jar) Unloaded->Constructed->Pre-initialized
	AWWayofTime{v1.3.3} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.3-4.jar) Unloaded->Constructed->Pre-initialized
	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.10.jar) Unloaded->Constructed->Pre-initialized
	Thaumcraft{4.2.3.5} [Thaumcraft] (Thaumcraft-1.7.10-4.2.3.5.jar) Unloaded->Constructed->Pre-initialized
	Botania{r1.6-192} [Botania] (Botania r1.6-192.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Builders{7.0.8} [BC Builders] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Energy{7.0.8} [BC Energy] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Robotics{7.0.8} [BC Robotics] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Silicon{7.0.8} [BC Silicon] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Transport{7.0.8} [BC Transport] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	BuildCraft|Factory{7.0.8} [BC Factory] (buildcraft-7.0.8.jar) Unloaded->Constructed->Pre-initialized
	CarpentersBlocks{3.3.6} [Carpenter's Blocks] (Carpenter's Blocks v3.3.6 - MC 1.7.10.jar) Unloaded->Constructed->Pre-initialized
	ChickenChunks{1.3.4.19} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.19-universal.jar) Unloaded->Constructed->Pre-initialized
	Railcraft{9.6.1.0} [Railcraft] (Railcraft_1.7.10-9.6.1.0.jar) Unloaded->Constructed->Pre-initialized
	TwilightForest{2.3.7} [The Twilight Forest] (twilightforest-1.7.10-2.3.7.jar) Unloaded->Constructed->Pre-initialized
	ForgeMultipart{1.2.0.344} [Forge Multipart] (ForgeMultipart-1.7.10-1.2.0.344-universal.jar) Unloaded->Constructed->Pre-initialized
	chisel{2.3.10.37} [Chisel 2] (Chisel2-2.3.10.37.jar) Unloaded->Constructed->Pre-initialized
	CompactSolars{4.4.39.315} [Compact Solar Arrays] (CompactSolars-1.7.10-4.4.39.315-universal.jar) Unloaded->Constructed->Pre-initialized
	ComputerCraft{1.73} [ComputerCraft] (ComputerCraft1.73.jar) Unloaded->Constructed->Pre-initialized
	CustomMainMenu{1.5} [Custom Main Menu] (CustomMainMenu-MC1.7.10-1.5.jar) Unloaded->Constructed->Pre-initialized
	iChunUtil{4.2.2} [iChunUtil] (iChunUtil-4.2.2.jar) Unloaded->Constructed->Pre-initialized
	Doors{4.0.1} [Doors] (Doors-4.0.1.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded{1.7.10R2.8.1B1} [MineFactory Reloaded] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	EnderIO{1.7.10-2.2.8.381} [Ender IO] (EnderIO-1.7.10-2.2.8.381.jar) Unloaded->Constructed->Pre-initialized
	EnderStorage{1.4.7.36} [EnderStorage] (EnderStorage-1.7.10-1.4.7.36-universal.jar) Unloaded->Constructed->Pre-initialized
	EnderZoo{1.7.10-1.0.11.28} [Ender Zoo] (EnderZoo-1.7.10-1.0.11.28.jar) Unloaded->Constructed->Pre-initialized
	EE3{0.3.507} [Equivalent Exchange 3] (EquivalentExchange3-1.7.10-0.3.507.jar) Unloaded->Constructed->Pre-initialized
	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) Unloaded->Constructed->Pre-initialized
	TConstruct{1.7.10-1.8.5.build957} [Tinkers' Construct] (TConstruct-1.7.10-1.8.5.jar) Unloaded->Constructed->Pre-initialized
	ExtraUtilities{1.2.6} [Extra Utilities] (extrautilities-1.2.6.jar) Unloaded->Constructed->Pre-initialized
	factorization.notify{1.0} [Factorization Notification System] (Factorization-1.7.10-0.8.88.1.jar) Unloaded->Constructed->Pre-initialized
	factorization{1.7.10-0.8.88.1} [Factorization] (Factorization-1.7.10-0.8.88.1.jar) Unloaded->Constructed->Pre-initialized
	factorization.misc{1.7.10-0.8.88.1} [Factorization Miscellaneous Nonsense] (Factorization-1.7.10-0.8.88.1.jar) Unloaded->Constructed->Pre-initialized
	FastCraft{1.21} [FastCraft] (fastcraft-1.21.jar) Unloaded->Constructed->Pre-initialized
	fastleafdecay{1.0} [Fast Leaf Decay] (FastLeafDecay-1.7.10-1.0.jar) Unloaded->Constructed->Pre-initialized
	FlatSigns{2.1.0.19} [Flat Signs] (FlatSigns-1.7.10-universal-2.1.0.19.jar) Unloaded->Constructed->Pre-initialized
	funkylocomotion{1.0} [Funky Locomotion] (funky-locomotion-1.7.10-beta-5c.jar) Unloaded->Constructed->Pre-initialized
	GardenCore{1.7.10-1.6.3} [Garden Core] (GardenStuff-1.7.10-1.6.3.jar) Unloaded->Constructed->Pre-initialized
	GardenContainers{1.7.10-1.6.3} [Garden Containers] (GardenStuff-1.7.10-1.6.3.jar) Unloaded->Constructed->Pre-initialized
	GardenStuff{1.7.10-1.6.3} [Garden Stuff] (GardenStuff-1.7.10-1.6.3.jar) Unloaded->Constructed->Pre-initialized
	GardenTrees{1.7.10-1.6.3} [Garden Trees] (GardenStuff-1.7.10-1.6.3.jar) Unloaded->Constructed->Pre-initialized
	MagicBees{1.7.10-2.3.1} [Magic Bees] (magicbees-1.7.10-2.3.1.jar) Unloaded->Constructed->Pre-initialized
	gendustry{1.4.13.94} [GenDustry] (gendustry-1.4.13.94-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	GraviSuite{1.7.10-2.0.3} [Graviation Suite] (GraviSuite-1.7.10-2.0.3.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft{2.1.0a} [Growthcraft] (growthcraft-core-1.7.10-2.1.0a.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft|Cellar{2.1.0a} [Growthcraft Cellar] (growthcraft-cellar-1.7.10-2.1.0a.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft|Apples{2.1.0} [Growthcraft Apples] (growthcraft-apples-1.7.10-2.1.0.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft|Bamboo{2.1.0} [Growthcraft Bamboo] (growthcraft-bamboo-1.7.10-2.1.0.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft|Grapes{2.1.0} [Growthcraft Grapes] (growthcraft-grapes-1.7.10-2.1.0.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft|Hops{2.1.0} [Growthcraft Hops] (growthcraft-hops-1.7.10-2.1.0.jar) Unloaded->Constructed->Pre-initialized
	Growthcraft|Rice{2.1.0a} [Growthcraft Rice] (growthcraft-rice-1.7.10-2.1.0a.jar) Unloaded->Constructed->Pre-initialized
	gud_thaumicnei{1.2.3} [Thaumic NEI] (gud_thaumicnei-1.2.3.jar) Unloaded->Constructed->Pre-initialized
	Hats{4.0.1} [Hats] (Hats-4.0.1.jar) Unloaded->Constructed->Pre-initialized
	HatStand{4.0.0} [HatStand] (HatStand-4.0.0.jar) Unloaded->Constructed->Pre-initialized
	HungerInPeace{1.0.0} [Hunger In Peace] (HungerInPeace-mc1.7.10-1.0.0.jar) Unloaded->Constructed->Pre-initialized
	IC2NuclearControl{2.1.2a} [Nuclear Control 2] (IC2NuclearControl-2.1.2a.jar) Unloaded->Constructed->Pre-initialized
	IGWMod{1.1.3-18} [In-Game wiki Mod] (IGW-Mod-1.7.10-1.1.3-18-universal.jar) Unloaded->Constructed->Pre-initialized
	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) Unloaded->Constructed->Pre-initialized
	IronChest{6.0.62.742} [Iron Chest] (ironchest-1.7.10-6.0.62.742-universal.jar) Unloaded->Constructed->Pre-initialized
	JABBA{1.2.1} [JABBA] (Jabba-1.2.1a_1.7.10.jar) Unloaded->Constructed->Pre-initialized
	journeymap{@JMVERSION@} [JourneyMap] (journeymap-1.7.10-5.1.0rc3-unlimited.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.1B1} [MFR Compat: Applied Energistics] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatAtum{1.7.10R2.8.1B1} [MFR Compat: Atum] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatBackTools{1.7.10R2.8.1B1} [MFR Compat: BackTools] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.1B1} [MFR Compat: BuildCraft] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatChococraft{1.7.10R2.8.1B1} [MFR Compat: Chococraft] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.1B1} [MFR Compat: ExtraBiomes] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatForestry{1.7.10R2.8.1B1} [MFR Compat: Forestry] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.1B1} [MFR Compat: ForgeMicroblock] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatIC2{1.7.10R2.8.1B1} [MFR Compat: IC2] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	Mystcraft{0.11.11.00} [Mystcraft] (mystcraft-1.7.10-0.11.11.00.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatMystcraft{1.7.10R2.8.1B1} [MFR Compat: Mystcraft] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MrTJPCoreMod{1.0.8.16} [MrTJPCore] (MrTJPCore-1.7.10-1.0.8.16-universal.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Core{4.6.2.82} [ProjectRed] (ProjectRed-1.7.10-4.6.2.82-Base.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Exploration{4.6.2.82} [ProjectRed-Exploration] (ProjectRed-1.7.10-4.6.2.82-World.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatProjRed{1.7.10R2.8.1B1} [MFR Compat ProjectRed] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.1B1} [MFR Compat: Railcraft] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.1B1} [MFR Compat: Sufficient Biomes] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.1B1} [MFR Compat: Thaumcraft] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.1B1} [MFR Compat: Thermal Expansion] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatTConstruct{1.7.10R2.8.1B1} [MFR Compat: Tinkers' Construct] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.1B1} [MFR Compat: TwilightForest] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineFactoryReloaded|CompatVanilla{1.7.10R2.8.1B1} [MFR Compat: Vanilla] (MineFactoryReloaded-[1.7.10]2.8.1B1-113.jar) Unloaded->Constructed->Pre-initialized
	MineTweaker3{3.0.9B} [MineTweaker 3] (MineTweaker3-1.7.10-3.0.9C.jar) Unloaded->Constructed->Pre-initialized
	numina{0.4.0.101} [Numina] (Numina-0.4.0.101.jar) Unloaded->Constructed->Pre-initialized
	powersuits{0.11.0.181} [MachineMuse's Modular Powersuits] (ModularPowersuits-0.11.0.181.jar) Unloaded->Constructed->Pre-initialized
	Morph{0.9.1} [Morph] (Morph-Beta-0.9.1.jar) Unloaded->Constructed->Pre-initialized
	Morpheus{1.7.10-1.6.4} [Morpheus] (Morpheus-1.7.10-1.6.4.jar) Unloaded->Constructed->Pre-initialized
	cfm{3.4.7} [§9MrCrayfish's Furniture Mod] (MrCrayfishFurnitureModv3.4.7(1.7.10).jar) Unloaded->Constructed->Pre-initialized
	NEIAddons{1.12.9.32} [NEI Addons] (neiaddons-1.12.9.32-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	NEIAddons|AppEng{1.12.9.32} [NEI Addons: Applied Energistics 2] (neiaddons-1.12.9.32-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	NEIAddons|Botany{1.12.9.32} [NEI Addons: Botany] (neiaddons-1.12.9.32-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	NEIAddons|Forestry{1.12.9.32} [NEI Addons: Forestry] (neiaddons-1.12.9.32-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	NEIAddons|CraftingTables{1.12.9.32} [NEI Addons: Crafting Tables] (neiaddons-1.12.9.32-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	NEIAddons|ExNihilo{1.12.9.32} [NEI Addons: Ex Nihilo] (neiaddons-1.12.9.32-mc1.7.10.jar) Unloaded->Constructed->Pre-initialized
	neiintegration{1.0.9} [NEI Integration] (NEIIntegration-MC1.7.10-1.0.9.jar) Unloaded->Constructed->Pre-initialized
	NotEnoughCodecs{0.3} [NotEnoughCodecs] (NotEnoughCodecs-1.7.10-0.3-snapshot-11.jar) Unloaded->Constructed->Pre-initialized
	notenoughkeys{@MOD_VERSION@} [NotEnoughKEys] (NotEnoughKeys-1.7.10-1.0.0b29.jar) Unloaded->Constructed->Pre-initialized
	ObsidiPlates{3.0.0.18} [ObsidiPlates] (ObsidiPlates-1.7.10-universal-3.0.0.18.jar) Unloaded->Constructed->Pre-initialized
	OpenMods{0.7.3} [OpenMods] (OpenModsLib-1.7.10-0.7.3.jar) Unloaded->Constructed->Pre-initialized
	OpenBlocks{1.4.3} [OpenBlocks] (OpenBlocks-1.7.10-1.4.3.jar) Unloaded->Constructed->Pre-initialized
	OpenPeripheralCore{1.1.1} [OpenPeripheralCore] (OpenPeripheralCore-1.7.10-1.1.1.jar) Unloaded->Constructed->Pre-initialized
	OpenPeripheral{0.3.1} [OpenPeripheralAddons] (OpenPeripheralAddons-1.7.10-0.3.1.jar) Unloaded->Constructed->Pre-initialized
	OpenPeripheralIntegration{0.2.2} [OpenPeripheralIntegration] (OpenPeripheralIntegration-1.7.10-0.2.2.jar) Unloaded->Constructed->Pre-initialized
	PneumaticCraft{1.8.0-82} [PneumaticCraft] (PneumaticCraft-1.7.10-1.8.0-82-universal.jar) Unloaded->Constructed->Errored
	PowerConverters{1.7.10-2.6} [Power Converters] (PowerConverters-1.7.10-2.6.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Transmission{4.6.2.82} [ProjectRed-Transmission] (ProjectRed-1.7.10-4.6.2.82-Integration.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Transportation{4.6.2.82} [ProjectRed-Transportation] (ProjectRed-1.7.10-4.6.2.82-Mechanical_beta.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Compatibility{4.6.2.82} [ProjectRed-Compatibility] (ProjectRed-1.7.10-4.6.2.82-Compat.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Integration{4.6.2.82} [ProjectRed-Integration] (ProjectRed-1.7.10-4.6.2.82-Integration.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Illumination{4.6.2.82} [ProjectRed-Illumination] (ProjectRed-1.7.10-4.6.2.82-Lighting.jar) Unloaded->Constructed->Pre-initialized
	ProjRed|Expansion{4.6.2.82} [ProjectRed-Expansion] (ProjectRed-1.7.10-4.6.2.82-Mechanical_beta.jar) Unloaded->Constructed->Pre-initialized
	RedstoneArsenal{1.7.10R1.1.0} [Redstone Arsenal] (RedstoneArsenal-[1.7.10]1.1.0-83.jar) Unloaded->Constructed->Pre-initialized
	simplyjetpacks{1.5.1} [Simply Jetpacks] (SimplyJetpacks-MC1.7.10-1.5.1.jar) Unloaded->Constructed->Pre-initialized
	SolarFlux{1.7.10-0.7} [Solar Flux] (SolarFlux-1.7.10-0.7.jar) Unloaded->Constructed->Pre-initialized
	StevesCarts{2.0.0.b18} [Steve's Carts 2] (StevesCarts2.0.0.b18.jar) Unloaded->Constructed->Pre-initialized
	StevesFactoryManager{A93} [Steve's Factory Manager] (StevesFactoryManagerA93.jar) Unloaded->Constructed->Pre-initialized
	StevesWorkshop{0.5.1} [Steve's Workshop] (StevesWorkshop-0.5.1.jar) Unloaded->Constructed->Pre-initialized
	foxlib{1.7.10-0.7.0} [FoxLib] (FoxLib-1.7.10-0.7.0.jar) Unloaded->Constructed->Pre-initialized
	Tails{1.7.10-1.3.1} [Tails] (Tails-1.7.10-1.3.1.jar) Unloaded->Constructed->Pre-initialized
	thaumicenergistics{0.8.10.5} [Thaumic Energistics] (thaumicenergistics-0.8.10.5.jar) Unloaded->Constructed->Pre-initialized
	ThermalDynamics{1.7.10R1.0.0} [Thermal Dynamics] (ThermalDynamics-[1.7.10]1.0.0-130.jar) Unloaded->Constructed->Pre-initialized
	TiCTooltips{1.2.4} [TiC Tooltips] (TiCTooltips-mc1.7.10-1.2.4.jar) Unloaded->Constructed->Pre-initialized
	TMechworks{0.2.14.100} [Tinkers' Mechworks] (TMechworks-1.7.10-0.2.14.100.jar) Unloaded->Constructed->Pre-initialized
	Translocator{1.1.1.14} [Translocator] (Translocator-1.7.10-1.1.1.14-universal.jar) Unloaded->Constructed->Pre-initialized
	VeinMiner{0.27.1_build.unknown} [Vein Miner] (VeinMiner-1.7.10_0.27.1.unknown.jar) Unloaded->Constructed->Pre-initialized
	VeinMinerModSupport{0.27.1_build.unknown} [Mod Support] (VeinMiner-1.7.10_0.27.1.unknown.jar) Unloaded->Constructed->Pre-initialized
	VersionChecker{1.1.8} [Version Checker] (VersionChecker-1.1.8.jar) Unloaded->Constructed->Pre-initialized
	WailaHarvestability{1.1.2} [Waila Harvestability] (WailaHarvestability-mc1.7.x-1.1.2.jar) Unloaded->Constructed->Pre-initialized
	witchery{0.24.1} [Witchery] (witchery-1.7.10-0.24.1.jar) Unloaded->Constructed->Pre-initialized
	WR-CBE|Core{1.4.1.9} [WR-CBE Core] (WR-CBE-1.7.10-1.4.1.9-universal.jar) Unloaded->Constructed->Pre-initialized
	WR-CBE|Addons{1.4.1.9} [WR-CBE Addons] (WR-CBE-1.7.10-1.4.1.9-universal.jar) Unloaded->Constructed->Pre-initialized
	WR-CBE|Logic{1.4.1.9} [WR-CBE Logic] (WR-CBE-1.7.10-1.4.1.9-universal.jar) Unloaded->Constructed->Pre-initialized
	McMultipart{1.2.0.344} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.2.0.344-universal.jar) Unloaded->Constructed->Pre-initialized
	aobd{2.6.3} [Another One Bites The Dust] (AOBD-2.6.3.jar) Unloaded->Constructed->Pre-initialized
	denseores{1.0} [Dense Ores] (denseores-1.6.2.jar) Unloaded->Constructed->Pre-initialized
	ForgeMicroblock{1.2.0.344} [Forge Microblocks] (ForgeMultipart-1.7.10-1.2.0.344-universal.jar) Unloaded->Constructed->Pre-initialized
	OpenModsLib class transformers: [stencil_patches:FINISHED],[movement_callback:FINISHED],[map_gen_fix:FINISHED],[gl_capabilities_hook:FINISHED],[player_render_hook:FINISHED]
	Class transformer null safety: all safe
	AE2 Version: stable rv2-stable-6 for Forge 10.13.2.1291
	CoFHCore: -[1.7.10]3.0.2-283
	Mantle Environment: DO NOT REPORT THIS CRASH! Unsupported mods in environment: optifine
	ThermalFoundation: -[1.7.10]1.0.0-87
	ThermalExpansion: -[1.7.10]4.0.1-182
	MineFactoryReloaded: -[1.7.10]2.8.1B1-113
	TConstruct Environment: Environment healthy.
	RedstoneArsenal: -[1.7.10]1.1.0-83
	ThermalDynamics: -[1.7.10]1.0.0-130
	List of loaded APIs: 
		* API_NEK (1.0.0) from NotEnoughKeys-1.7.10-1.0.0b29.jar
		* AppleCoreAPI (1.1.0) from AppleCore-mc1.7.10-1.1.0.jar
		* appliedenergistics2|API (rv2) from appliedenergistics2-rv2-stable-6.jar
		* Baubles|API (1.0.1.10) from Baubles-1.7.10-1.0.1.10.jar
		* BiomesOPlentyAPI (1.0.0) from BiomesOPlenty-1.7.10-2.1.0.1067-universal.jar
		* BotaniaAPI (51) from Botania r1.6-192.jar
		* BuildCraftAPI|blocks (1.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|blueprints (1.0) from bcapi-6.0.x.jar
		* BuildCraftAPI|boards (2.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|core (1.5) from buildcraft-7.0.8.jar
		* BuildCraftAPI|crops (1.1) from buildcraft-7.0.8.jar
		* BuildCraftAPI|events (1.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|facades (1.1) from buildcraft-7.0.8.jar
		* BuildCraftAPI|filler (4.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|fuels (2.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|gates (4.1) from buildcraft-7.0.8.jar
		* BuildCraftAPI|items (1.1) from buildcraft-7.0.8.jar
		* BuildCraftAPI|library (2.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|power (1.3) from buildcraft-7.0.8.jar
		* BuildCraftAPI|recipes (3.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|robotics (2.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|statements (1.0) from Railcraft_1.7.10-9.6.1.0.jar
		* BuildCraftAPI|tablet (1.0) from buildcraft-7.0.8.jar
		* BuildCraftAPI|tiles (1.2) from buildcraft-7.0.8.jar
		* BuildCraftAPI|tools (1.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* BuildCraftAPI|transport (2.0) from bcapi-6.0.x.jar
		* ChiselAPI (0.0.2) from Chisel2-2.3.10.37.jar
		* ChiselAPI|Carving (0.0.2) from Chisel2-2.3.10.37.jar
		* CoFHAPI (1.7.10R1.0.1) from Railcraft_1.7.10-9.6.1.0.jar
		* CoFHAPI|block (1.7.10R1.0.12) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHAPI|core (1.7.10R1.0.12) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHAPI|energy (1.7.10R1.0.2) from forestry_1.7.10-3.6.0.586-unstable.jar
		* CoFHAPI|fluid (1.7.10R1.0.5) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHAPI|inventory (1.7.10R1.0.12) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHAPI|item (1.7.10R1.0.4) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHAPI|modhelpers (1.7.10R1.0.4) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHAPI|tileentity (1.7.10R1.0.3) from EnderIO-1.7.10-2.2.8.381.jar
		* CoFHAPI|transport (1.7.10R1.0.4) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHAPI|world (1.7.10R1.0.12) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|audio (1.7.10R1.0.0B8) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHLib|gui (1.7.10R1.0.0B8) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHLib|gui|container (1.7.10R1.0.0B8) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHLib|gui|element (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|gui|element|listbox (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|gui|slot (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|inventory (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|render (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|render|particle (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|util (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|util|helpers (1.7.10R1.0.0B8) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHLib|util|position (1.7.10R1.0.0B8) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* CoFHLib|world (1.7.10R1.0.3B1) from CoFHCore-[1.7.10]3.0.2-283.jar
		* CoFHLib|world|feature (1.7.10R1.0.0B8) from CoFHLib-[1.7.10]1.0.3B1-163.jar
		* ComputerCraft|API (1.73) from ComputerCraft1.73.jar
		* ComputerCraft|API|FileSystem (1.73) from ComputerCraft1.73.jar
		* ComputerCraft|API|Lua (1.73) from ComputerCraft1.73.jar
		* ComputerCraft|API|Media (1.73) from ComputerCraft1.73.jar
		* ComputerCraft|API|Peripheral (1.73) from ComputerCraft1.73.jar
		* ComputerCraft|API|Redstone (1.73) from ComputerCraft1.73.jar
		* ComputerCraft|API|Turtle (1.73) from ComputerCraft1.73.jar
		* EE3-API|array (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EE3-API|core (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EE3-API|event (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EE3-API|exchange (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EE3-API|knowledge (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EE3-API|recipe (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EE3-API|util (1.2) from EquivalentExchange3-1.7.10-0.3.507.jar
		* EnderIOAPI (0.0.2) from EnderIO-1.7.10-2.2.8.381.jar
		* EnderIOAPI|Redstone (0.0.2) from EnderIO-1.7.10-2.2.8.381.jar
		* EnderIOAPI|Tools (0.0.2) from EnderIO-1.7.10-2.2.8.381.jar
		* factorization notification system (1.0) from Factorization-1.7.10-0.8.88.1.jar
		* ForestryAPI|apiculture (3.5.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|arboriculture (2.3.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|circuits (2.0.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|core (3.2.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|farming (1.1.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|food (1.1.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|fuels (2.0.1) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|genetics (3.3.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|hives (4.1.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|lepidopterology (1.1) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|mail (3.0.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|recipes (3.1.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|storage (3.0.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* ForestryAPI|world (1.1.0) from forestry_1.7.10-3.6.0.586-unstable.jar
		* GardenCoreAPI (1.0.0) from GardenStuff-1.7.10-1.6.3.jar
		* gendustryAPI (1.4.13.94) from gendustry-1.4.13.94-mc1.7.10.jar
		* GrowthcraftAPI|Cellar (2.0) from growthcraft-cellar-1.7.10-2.1.0a.jar
		* GrowthcraftAPI|Core (2.0) from growthcraft-core-1.7.10-2.1.0a.jar
		* IC2API (1.0) from advancedgenetics-1.7.10-1.5.7.jar
		* Mystcraft|API (0.1) from mystcraft-1.7.10-0.11.11.00.jar
		* NuclearControlAPI (v1.0.5) from IC2NuclearControl-2.1.2a.jar
		* OpenBlocks|API (1.0) from OpenBlocks-1.7.10-1.4.3.jar
		* OpenPeripheralApi (3.2) from OpenPeripheralCore-1.7.10-1.1.1.jar
		* PneumaticCraftApi (1.0) from PneumaticCraft-1.7.10-1.8.0-82-universal.jar
		* RailcraftAPI|bore (1.0.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|carts (1.3.1) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|core (1.3.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|crafting (1.0.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|electricity (1.6.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|events (1.0.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|fuel (1.0.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|helpers (1.1.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|items (1.0.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|locomotive (1.0.1) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|signals (1.3.0) from Railcraft_1.7.10-9.6.1.0.jar
		* RailcraftAPI|tracks (2.0.0) from Railcraft_1.7.10-9.6.1.0.jar
		* TailsAPI (1.1.1) from Tails-1.7.10-1.3.1.jar
		* Thaumcraft|API (4.2.2.0) from Railcraft_1.7.10-9.6.1.0.jar
		* VeinMinerApi (0.1) from VeinMiner-1.7.10_0.27.1.unknown.jar
		* WailaAPI (1.2) from Waila-1.5.10_1.7.10.jar
	Stencil buffer state: Function set: GL30, pool: forge, bits: 8

--- a/src/pneumaticCraft/common/thirdparty/computercraft/OpenComputers.java
+++ b/src/pneumaticCraft/common/thirdparty/computercraft/OpenComputers.java
@@ -1,5 +1,6 @@
 package pneumaticCraft.common.thirdparty.computercraft;
 
+import cpw.mods.fml.common.Optional;
 import li.cil.oc.api.Driver;
 import net.minecraft.block.Block;
 import net.minecraft.block.material.Material;
@@ -33,6 +34,13 @@ public class OpenComputers implements IThirdParty{
         if(!Loader.isModLoaded(ModIds.COMPUTERCRAFT)) {
             GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(droneInterface), true, " u ", "mp ", "iii", 'u', new ItemStack(Itemss.machineUpgrade, 1, ItemMachineUpgrade.UPGRADE_RANGE), 'm', Items.ender_pearl, 'p', Itemss.printedCircuitBoard, 'i', Names.INGOT_IRON_COMPRESSED));
         }
+        if(Loader.isModLoaded(ModIds.OPEN_COMPUTERS)) {
+            initializeDrivers();
+        }
+    }
+
+    @Optional.Method(modid = ModIds.OPEN_COMPUTERS)
+    private void initializeDrivers(){
         Driver.add(new DriverPneumaticCraft());
     }
 

